official/resnet/layer_test.py

```diff
@@ -41,14 +41,22 @@
 DATA_FORMAT = "channels_last"  # CPU instructions often preclude channels_first
 BATCH_SIZE = 32
 BLOCK_TESTS = [
-    dict(bottleneck=True, projection=True, version=1, width=8, channels=4),
```

Comment:
@karmel Behold the casualties of your verbosity!

Reply:
I shed a tear for the fallen.